content/app-dev/multicap/_index.en.md

```diff
@@ -5,5 +5,14 @@ weight: 4
 draft: false
 ---
 
-TBD
+Many of the examples we've covered in the documentation have been actors that make use of a single capability. For example, we often illustrate a number of topics by creating an actor that responds to HTTP server requests with some equivalent of "Hello world!" This is just the tip of the iceberg! The real power of wasmCloud shines when you start plugging in multiple actor interfaces to a single actor.
 
+### Combining Key-Value and HTTP Server
+
+The [kvcounter](https://github.com/wasmCloud/examples/tree/main/kvcounter) example provides an illustration of both handling HTTP requests from a web server and read/write operations against a key-value store.

```

Comment:
The only thing I noticed is that this link doesn't go anywhere, do we have some examples planned for the `wasmCloud` repo?

Reply:
this will go somewhere once the examples tree has been renamed from `master` to `main`

Reply:
Ah, I see now that it's the `examples` repo not `wasmcloud`. LGTM